src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,129 @@
+package frc.robot.subsystems.arm;
+
+//importing libraries
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+// extend SubsystemBase class into the Arm class?
+public class Arm {
+
+    // making variable under the CANSparkMax class
+    private CANSparkMax leftMotor;
+    private CANSparkMax rightMotor;
+
+    //variables to set limits, speeds, and ramp-rate
+    private final float REVERSE_LIMIT = (float) 0;
```

Comment:
change this from a cast of 0 to 0.0f
eg
private final float  REVERSE_LIMIT = 0.0f;